### DISCUSSION TITLE
Cursor size too small (mouse pointer)

### DISCUSSION THREAD

**Discussion by applemuncy** [ltsp/ltsp#382] [Q&A] [answered]
I have ltsp up and running mate as recommended except I did not uninstall mate-hud. 
It is working well so time to tweak the  small details.
What do the rest of you do to make the cursor (mouse pointer) larger?

I read elsewhere that I'm to use "Universal Access" and select "cursor size" but no "cursor size".

**Comment by alkisg** [comment, marked as answer]
You might want to ask in the MATE forums for that one, e.g.
https://ubuntu-mate.community/c/support/10

**Reply by applemuncy** [reply]
Thank you @alkisg You pointer me to the right place.  I found the answer right away.  And thank for this great project.

"You can enlarge the mouse cursor in Appearance settings → Theme tab → Customize… button → Pointer tab → Size (at the bottom of the window)."
